test/edu/ucsb/nceas/MCTestCase.java

```diff
@@ -405,165 +360,164 @@ protected String getTestEmlDoc(String title, String emlVersion) {
         String testDocument = "";
 
         String header;
-		if (emlVersion == EML2_0_1) {
-			header = testEml_201_Header;
-		} else if (emlVersion == EML2_1_0) {
-			header = testEml_210_Header;
-		} else { // if (emlVersion == EML2_1_1) {
-			header = testEml_211_Header;
-		}
-        
+        if (emlVersion == EML2_0_1) {
+            header = testEml_201_Header;
+        } else if (emlVersion == EML2_1_0) {
+            header = testEml_210_Header;
+        } else { // if (emlVersion == EML2_1_1) {
+            header = testEml_211_Header;
+        }
+
         testDocument += header;
-        
+
         // if this is an EML 2.1.0 or later document, the document level access is
         // before the dataset element.
         if (emlVersion == EML2_1_0 || emlVersion == EML2_1_1) {
-        	testDocument += getAccessBlock("public", true, true, false, false, false);
+            testDocument += getAccessBlock("public", true, true, false, false, false);
         }
         testDocument += "<dataset scope=\"document\"><title>" + title + "</title>"
-				+ testEmlCreatorBlock;
+                + testEmlCreatorBlock;
 
         testDocument += testEmlContactBlock;
-        
+
         // if this is an EML 2.0.1 or earlier document, the document level access is
         // inside the dataset element.
         if (emlVersion != EML2_1_0 && emlVersion != EML2_1_1) {
-        	testDocument += getAccessBlock("public", true, true, false, false, false);
+            testDocument += getAccessBlock("public", true, true, false, false, false);
+        }
+        testDocument += "</dataset>";
+        testDocument += "</eml:eml>";
+
+        return testDocument;
+    }
+
+    /**
+     * This function returns a valid eml document with no access rules
+     */
+    protected String getTestEmlDoc(String title, String emlVersion, String inlineData1,
```

Comment:
Another rabbit hole, but it would be nice to externalize test docs as resources and then iterate over them.

Reply:
(Agreed and noted. Not familiar with this file at all - only edits I made to this file were in the static init block up top, to change how properties are loaded for tests at runtime)